### ISSUE TITLE
Create the Feed, Water, Rest UJMs from USMs 

### ISSUE THREAD

**Issue by holtzmak** [holtzmak/Humane-Transport#49]
Using the USM from #42, create the corresponding User Journey Maps (UJMs) using the structure from #47.

You'll likely notice that the UJM structure doesn't match what you see the UJM as, so if we need to make changes to the GUI structure, talk to the assignee of #47.

**Comment by SanaKhan1997** [comment]
Thanks! @MummenRider I've made few comments in the [draft](https://docs.google.com/document/d/1uSPQPbEhqp_qIphvbAtQnNQNZ38YhX5PDIv3wK8SnEE/edit?usp=sharing) other than those it looks great.

**Comment by MummenRider** [comment]
I created FWR UJM using Figma [here](https://www.figma.com/file/GTe2plpJMejL3I4KCSIudt/UJM-FWR?node-id=0%3A1).

**Comment by holtzmak** [comment]
Your Figma template, while colorful, is very similar to the NN/g template. I've mentioned this before, but I'm personally fine with using both for UJMs. 

You will need to merge this PDF into the codebase at some point (until we find another way to store documentation out of the codebase), so best to make a PR in the future for these things. Is this the only UJM you're making for FWR?

One thing to mention, and is probably related to [your comment here](https://github.com/holtzmak/Humane-Transport/pull/58#issuecomment-727604026), I'm not sure how to feel about the "emotion line" or w/e that you've made. e.g. "Wow, filling out that form was easy!". These sort of emotion points are obviously what we want to bring out from transporters and it feels somewhat cheesy to have to write down how we want them to feel.

I like the doing, thinking, saying of the NN/g template: it's goal-focused emotion: when we brainstorm, we have to put ourselves into the shoes of the transporter and do what they would do - and this makes us ask questions about how the transporter may be feeling when doing an action and that drives us to make particular design decisions. I think this performs better than writing about the user's experience with the app. Sana had mentioned on the same PR linked above that the phases of the UJM will emulate where and when emotions arise.

**Comment by MummenRider** [comment]
> Your Figma template, while colorful, is very similar to the NN/g template. I've mentioned this before, but I'm personally fine with using both for UJMs.
> 
> You will need to merge this PDF into the codebase at some point (until we find another way to store documentation out of the codebase), so best to make a PR in the future for these things. Is this the only UJM you're making for FWR?
> 
> One thing to mention, and is probably related to [your comment here](https://github.com/holtzmak/Humane-Transport/pull/58#issuecomment-727604026), I'm not sure how to feel about the "emotion line" or w/e that you've made. e.g. "Wow, filling out that form was easy!". These sort of emotion points are obviously what we want to bring out from transporters and it feels somewhat cheesy to have to write down how we want them to feel.
> 
> I like the doing, thinking, saying of the NN/g template: it's goal-focused emotion: when we brainstorm, we have to put ourselves into the shoes of the transporter and do what they would do - and this makes us ask questions about how the transporter may be feeling when doing an action and that drives us to make particular design decisions. I think this performs better than writing about the user's experience with the app. Sana had mentioned on the same PR linked above that the phases of the UJM will emulate where and when emotions arise.

@holtzmak , it may be cheesy, but it's part of user's emotion and I think is important to show as part of their journey.
[Here ](https://www.nngroup.com/articles/analyze-customer-journey-map/)is an example created by nngroup.
I believe "doing" is already part of the UJM Figma I created. I also love the "thinking" and "saying", what I can do to improve the current Figma template is simply add those 2 into it. There is enough space to add those 2 there.

**Comment by holtzmak** [comment]
> @holtzmak , it may be cheesy, but it's part of user's emotion and I think is important to show as part of their journey.
Here is an example created by nngroup.
I believe "doing" is already part of the UJM Figma I created. I also love the "thinking" and "saying", what I can do to improve the current Figma template is simply add those 2 into it. There is enough space to add those 2 there.

Having a look at that template you shared, I noticed that some of the feeling and sayings in the document are very similar to what I'd expect feedback from testers when faced with our prototypes. I think it's too early for us to decide what people using the application want - I think we can use their feedback to reflect in this way vs. assume things now. 

I wouldn't mind experimenting with the UJMs: we use both the Figma and the NN/g templates, and when we get to the prototype testing and find that more changes are requested of everything else vs. FWR, that could be a signifier of us needed to do the work of assuming emotions. Of course, FWR is a portion of the overall ATR so it might not have much statistically correct impact like I assume.
